### PR TITLE
6.5: Uploaded rt patch

### PR DESCRIPTION
**Pull request by MichaelDeets**
Based on [v6.5-rc7-rt4](https://git.kernel.org/pub/scm/linux/kernel/git/rt/linux-rt-devel.git/tag/?h=v6.5-rc7-rt4).
Removed patches related to ARM.
Removed patches already applied by EEVDF and presumably TT, otherwise will fail during patching.
Must be applied after EEVDF/TT, hence the naming convention being `0002` and not `0001`

### CODE REVIEW

**Comment by ventureoo** [comment]
Looks good for me, build was also fine. But note that it would be nice if you included in your patch a commit (https://github.com/CachyOS/linux/commit/ddab46a8132224a5a114a9da1a29c204262ea7d5) to disable the CONFIG_EXPERT dependency, as it greatly simplifies config maintenance for us.

**Comment by ptr1337** [comment]
Please add the CONFIG_EXPERT removal and rename the patch to "0001-rt.patch" so that it will fit to our PKGBUILDs.
If it does work with EEVDF thats great. Then we can use for the RT kernel also EEVDF.

**Comment by MichaelDeets** [comment]
I've removed CONFIG_EXPERT requirement and renamed the patch.

To work with EEVDF I simply removed a few conflicting patch entries. One could apply the unmodified RT.patch and change EEVDF to work with it, but I found this method easier.

File: 6.5/misc/0001-rt.patch [hunk 1]
```diff
@@ -0,0 +1,7029 @@
...
++
+ config PREEMPT_NONE_BUILD
+ 	bool
+ 
+@@ -69,7 +75,7 @@ config PREEMPT
+ 
+ config PREEMPT_RT
+ 	bool "Fully Preemptible Kernel (Real-Time)"
+-	depends on EXPERT && ARCH_SUPPORTS_RT
++	depends ARCH_SUPPORTS_RT
```
Comment (ventureoo) [inline 6.5/misc/0001-rt.patch:2773]:
```suggestion
+	depends on ARCH_SUPPORTS_RT
```
You missed the ``on`` statement

**Review by ventureoo — APPROVED** [review]

**Pull request merged**